src/backend/storeManagers/hyperplay/games.ts

```diff
@@ -101,7 +101,13 @@ export const isGameAvailable = (appName: string) => {
   }
 
   if (hpGameInfo.install && hpGameInfo.install.executable) {
-    const { executable } = getExecutableAndArgs(hpGameInfo.install.executable)
+    let { executable } = getExecutableAndArgs(hpGameInfo.install.executable)
+
+    // on linux and mac replace backslashes with forward slashes on executable
+    if (!isWindows) {
+      executable = executable.replace(/\\/g, '/')
```

Comment:
can we use `path.normalize()` as per @jiyuu-jin 's comment

Reply:
cool, will change then

Reply:
This method doesnt work and the path keeps using backslashes, so we need to keep with the current solution